conf/cassandra.yaml

```diff
@@ -2368,3 +2368,104 @@ drop_compact_storage_enabled: false
+#  # This information is useful so the scheduler is absolutely sure that the node is indeed removed from the ring, and then it can adjust the repair schedule accordingly.
+#  # So, the duration in this config determinses for how long deleted host's information is kept in the scheduler's metadata.
+#  history_clear_delete_hosts_buffer_interval: 2h
+#  # Configuration specific to each repair type. Options: full, incremental, and preview_repaired.
+#  repair_type_overrides:
+#    full:
+#      # Enable/Disable full auto repair
+#      enabled: false
+#      # Minimum time in hours between repairing the same node again. This is useful for extremely tiny clusters, say 5 nodes, which finishes
+#      # repair quicly. The default is 24 hours. This means that if the scheduler finishes one round on all the nodes in < 24 hours. On a given node it won’t start a new repair round
+#      # until the last repair conducted on a given node is < 24 hours.
+#      min_repair_interval: 24h
+#      # Repair table by table if this is set to 'false'. If it is 'true', then repair all the tables in a keyspace in one go.
+#      repair_by_keyspace: false
+#      # Number of repair threads to run for a given invoked Repair Job.
+#      # Once the scheduler schedules one repair session, then howmany threads to use inside that job will be controlled through this parameter.
+#      # This is similar to -j for repair options for the nodetool repair command.
+#      number_of_repair_threads: 1
+#      # Threshold to skip a table if it has too many sstables. The default is 10000. This means, if a table on a node has 10000 or more SSTables, then that table will be skipped.
+#      # This is to avoid penalizing good tables (neighbors) with an outlier.
+#      sstable_upper_threshold: 10000
+#      # Max time for repairing one table on a given node, if exceeded, skip the table.
+#      # Let's say there is a Cassandra cluster in that there are 10 tables belonging to 10 different customers.
+#      # Out of these 10 tables, 1 table is humongous. Repairing this 1 table, say, takes 5 days, in the worst case, but others could finish in just 1 hour.
+#      # Then we would penalize 9 customers just because of one bad actor, and those 9 customers would ping an operator and would require a lot of back-and-forth manual interventions, etc.
+#      # So, the idea here is to penalize the outliers instead of good candidates. This can easily be configured with a higher value if we want to disable the functionality.
+#      table_max_repair_time: 6h
+#      # This is useful if you want to completely avoid running repairs in one or more data centers. By default, it is empty, i.e., the framework will repair nodes in all the datacenters.
+#      # If you want to avoid repairing nodes in one or more data centers, then you can specify the data centers in this list.
+#      ignore_dcs: []
+#      # Set this 'true' if AutoRepair should repair only the primary ranges owned by this node; else, 'false'.
```

Comment:
I can't think of why someone would want to set this to false, especially since ultimately every replica submits repairs.  Maybe we should make it more clear when one should consider setting it to false?  (Can't think of a reason off top of my head)

Reply:
Just added a reason.